.dvc/config

```diff
@@ -1,4 +1,6 @@
 [core]
-    remote = sawt
-['remote "sawt"']
+    remote = myremote
+['remote "myremote"']
     url = gs://faiss
+    projectname = the-great-inquirer
```

Comment:
@ayyubibrahimi do these values matter? should this be `sawt`?

Reply:
Making a note to fix this.

Reply:
@marvinmarnold this references the cloud project name. Holding off on changing this until we decide to change all references.

Reply:
Ah, gotcha. Ya, let's keep that for now. We'll have to change a bunch of small things otherwise.